1-exercises/B-while-loop/exercise.js

```diff
@@ -6,9 +6,13 @@
 */
 
 function evenNumbers(n) {
-    // TODO
+  let i = 0;
+  while (i < n) {
+    console.log(i * 2);
```

Comment:
The instructions are asking for a comma-separated string to be logged out. How can you collect all the results from the loop then log them out as a string, after the loop has finished?

Reply:
Hi Jon,
Thanks for your time reviewing my pr.
As I understood, we need to use .push to a new variable which should be an array.
Then this will return an array of numbers then by useing .toString will give us the resault :)